src/style/main.scss

```diff
@@ -2,3 +2,8 @@
 @use 'header.scss';
 @use 'footer.scss';
 @use 'search-section.scss';
+
+* {
```

Comment:
It should be at the very beginning

Reply:
* {
  box-sizing: border-box;
}

@use 'colors.scss';
@use 'header.scss';
@use 'footer.scss';
@use 'search-section.scss';   

Error: @use rules must be written before any other rules.
  ╷
5 │ @use 'colors.scss';

Reply:
Why not output my style to a separate file? For example _core.scss?